.github/workflows/push_pr.yaml

```diff
@@ -3,16 +3,20 @@ name: Fission CI
 on:
   push:
     branches:
-      - master
+      - master   
   pull_request:
     branches:
       - master
   workflow_dispatch:
-
 jobs:
   # Job to run change detection
   integration-test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest , ubuntu-20.04, ubuntu-18.04] 
+        kindimage : [ 'kindest/node:v1.19.11@sha256:7664f21f9cb6ba2264437de0eb3fe99f201db7a3ac72329547ec4373ba5f5911','kindest/node:v1.20.7@sha256:e645428988191fc824529fd0bb5c94244c12401cf5f5ea3bd875eb0a787f0fe9',  'kindest/node:v1.21.1@sha256:fae9a58f17f18f06aeac9772ca8b5ac680ebbed985e266f711d936e91d113bad' ]
```

Comment:
Kind images should we mention in simple format?

```
kindimage : [ 'kindest/node:v1.19.11', 'kindest/node:v1.20.7', 'kindest/node:v1.21.1' ]
```

Any reason for choosing different OS ? Can we just use `ubuntu-latest` instead ?


Also I think different kubernets version wont take effect as below kind cluster step has kind version hardcoded.

```
      - name: Kind Clutser
        uses: engineerd/setup-kind@v0.5.0
        with:
          config: kind.yaml
```
Reference kind.yaml,
https://github.com/fission/fission/blob/3f99b483cb65fd0d01a6591da7d626ccf6c77bcf/kind.yaml#L11

Reply:
Hi @sanketsudake, 
1- as per https://github.com/kubernetes-sigs/kind/releases it is advised to use exact images including digest.
2- thanks sanket, looks like missed the change adding now.